client/src/components/Related/Carousel/Card.jsx

```diff
@@ -9,8 +9,6 @@ import { faStar } from '@fortawesome/free-solid-svg-icons';
 import { solid, thin, regular, brands, icon } from '@fortawesome/fontawesome-svg-core/import.macro';
 import whiteBackground from '/client/src/components/assets/images/whiteBackground.jpg';
 import outfit from '/client/src/components/assets/outfit.jsx'
-// import product from "../../../../../server/exampleData/product.json"
```

Comment:
Sorry I didn't fix these paths when I did my initial revision pass on the file pathing.